django/contrib/admin/static/admin/css/changelists.css

```diff
@@ -148,10 +148,16 @@
     border-bottom: none;
 }
 
-#changelist-filter h3 {
+#changelist-filter details summary {
```

Comment:
I followed this guide to fix details/summary styles: https://css-tricks.com/two-issues-styling-the-details-element-and-how-to-solve-them/